app/models/graph/types/planet.rb

```diff
@@ -0,0 +1,20 @@
+module Graph
+  module Types
+    Planet = GraphQL::ObjectType.define do
+      name "Planet"
+      description "A planet in the StarWars universe"
+
+      field :id, types.ID
+
+      field :name, types.String
```

Comment:
We might want to eventually add descriptions for all these fields. Most of them are documented here: http://graphql.org/swapi-graphql/